AppDB/appscale/datastore/cassandra_env/cassandra_interface.py

```diff
@@ -77,6 +78,12 @@
 # The size in bytes that a batch must be to use the batches table.
 LARGE_BATCH_THRESHOLD = 5 << 10
 
+# Entities have a .78% chance of getting the scatter property. Their keys
+SCATTER_CHANCE = .0078
```

Comment:
isn't 0.78% 0.000078?

Reply:
Representing a percentage as a decimal just requires dividing by 100.

Reply:
had a feeling coffee didn't kick in for that comment.